include/revng/ABI/FunctionType/Support.h

```diff
@@ -39,6 +39,9 @@ inline constexpr uint64_t paddedSizeOnStack(uint64_t RealSize,
                                             uint64_t RegisterSize) {
   revng_assert(llvm::isPowerOf2_64(RegisterSize));
 
+  if (RealSize == 0)
+    return RealSize;
```

Comment:
I added this check because I was definitely hitting this branch somewhere.

But, thinking on it now, I was probably due to `void` arguments added by PE importer. Now that @dwarfyresearch fixed those, this could probably became an assert.